addons/website/views/snippets/s_features_wave.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_features_wave" name="Features Wave">
+    <section class="s_features_wave o_cc o_cc5 pt64 pb64" data-oe-shape-data="{'shape':'web_editor/Wavy/11', 'showOnMobile':true}">
+        <div class="o_we_shape o_web_editor_Wavy_11 o_shape_show_mobile"/>
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-12 pb-24">
+                    <h3 style="text-align: center;">Everything you need</h3>
+                    <p class="lead" style="text-align: center;">List and describe the key features of your solution or service.</p>
+                </div>
+                <div class="col-lg-4 pt24 pb24">
+                    <i class="fa fa-thumbs-up d-block mx-auto rounded fs-5 bg-o-color-1"/>
+                    <br/>
+                    <h5 style="text-align: center;">Reliability</h5>
+                    <p class="o_small" style="text-align: center;">Consistent performance and uptime ensure efficient, reliable service with minimal interruptions and quick response times.</p>
+                </div>
+                <div class="col-lg-4 pt24 pb24">
+                    <i class="fa fa-trophy d-block mx-auto rounded fs-5 bg-o-color-1"/>
+                    <br/>
+                    <h5 style="text-align: center;">Performance</h5>
+                    <p class="o_small" style="text-align: center;">Speed and efficiency ensure tasks are completed quickly and resources are used optimally, enhancing productivity and satisfaction.</p>
+                </div>
+                <div class="col-lg-4 pt24 pb24">
+                    <i class="fa fa-line-chart d-block mx-auto rounded fs-5 bg-o-color-1"/>
```

Comment:
You deviated from the mockup because it's not possible to set a border, right?
| mockup | this PR |
|--------|--------|
| <img width="715" alt="image" src="https://github.com/user-attachments/assets/716e4e8f-1348-4344-a73a-301ac44f9d15"> | <img width="623" alt="image" src="https://github.com/user-attachments/assets/e3d5c5a0-f252-46c4-b251-bfa6338ee24e"> | 

I believe we should give users the option to set a border on icons, especially since they already have the ability to set a background. Adding something like the following in `snippets.xml` should work nicely 😉
```xml
<div data-js="FontawesomeToolsWebsite" data-selector="span.fa, i.fa" data-exclude="[data-oe-xpath]">
    <t t-call="website.snippet_options_border_line_widgets">
        <t t-set="label">Border</t>
        <t t-set="with_bs_class" t-value="True"/>
    </t>
</div>
```

Reply:
I thought it was out of scope but ok, added a comit

Also adapter the sizings a bit. It's not possible to have the same icon size than in the Figma. It's either 16px or 32px :(